### DISCUSSION TITLE
Want to buy some coffee...

### DISCUSSION THREAD

**Discussion by boettchs** [will-stone/browserosaurus#467] [General]
I dismissed the "coffee" blurb at the first opening.
I love this app, and want to buy some caffeine for you!
How does I doos it?

Scott

**Comment by will-stone** [comment]
Hi @boettchs Thank you for using Browserosaurus and wanting to support my open source projects. There's "Sponsor" link above, where you can either use GitHub Sponsors or there's a link to Buy Me a Coffee.

The welcome message in Browserosaurus is timed to pop back up after a week, if you select "Maybe Later". Or if you select "Buy Me a Coffee" it'll never show again.

Thanks again,

Will.

**Reply by boettchs** [reply]
Thanks, @will-stone !!  I swear I looked there, but I see it now!  

Scott

**Comment by susodapop** [comment]
There's a small link format issue on your buymeacoffee success modal: 
<img width="512" alt="CleanShot 2021-12-22 at 10 32 00@2x" src="https://user-images.githubusercontent.com/17067911/147124986-7de19f2a-849f-40a0-939e-a5e14065c596.png">

### [Git](#)Hub should be [Github](#)

**Reply by will-stone** [reply]
Thanks! 🙂

**Comment by boettchs** [comment]
Hey @will-stone - I went from never getting the "Coffee" prompt to getting it a lot.  Now, I don't mind buying more coffee, but is there a way to nix that after indeed buying some coffee :)

**Reply by will-stone** [reply]
Hi @boettchs Sorry to hear that. Can you please elaborate? Is it every time you open it? How many times in the past week etc.? As long as you click “buy me a coffee” and then cancel the picker, it shouldn’t show again (unless I break an update! 🤭)

**Reply by boettchs** [reply]
I'll have to take more careful notes...not sure how often, but I've seen it ~3 times in the last 7-10 days ¯\\_(ツ)_/¯ 
Not a big deal, and it's possible that using Monterey beta may be the reason...

**Reply by will-stone** [reply]
Ah, it may just be because of the recent update. I had to change the config that kept track of whether you'd already clicked "buy me a coffee". 🤞 you shouldn't see it again.
